diff --git a/node_modules/@azesmway/react-native-unity/ios/RNUnityView.mm b/node_modules/@azesmway/react-native-unity/ios/RNUnityView.mm
index a4caa74..1b9994e 100644
--- a/node_modules/@azesmway/react-native-unity/ios/RNUnityView.mm
+++ b/node_modules/@azesmway/react-native-unity/ios/RNUnityView.mm
@@ -84,8 +84,14 @@ - (void)initUnityModule {
 - (void)layoutSubviews {
    [super layoutSubviews];

+   if(![self unityIsInitialized]) {
+      [self initUnityModule];
+   }
+
    if([self unityIsInitialized]) {
       self.ufw.appController.rootView.frame = self.bounds;
+      self.ufw.appController.rootView.opaque = NO;
+      self.ufw.appController.rootView.backgroundColor = [UIColor clearColor];
       [self addSubview:self.ufw.appController.rootView];
    }
 }
